CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ option(WITH_DRWAVAUDIOIMPORTER "Build DrWavAudioImporter plugin" OFF)
 option(WITH_FAAD2AUDIOIMPORTER "Build Faad2AudioImporter plugin" OFF)
 cmake_dependent_option(WITH_FREETYPEFONT "Build FreeTypeFont plugin" OFF "NOT WITH_HARFBUZZFONT" ON)
 option(WITH_HARFBUZZFONT "Build HarfBuzzFont plugin" OFF)
+option(WITH_ICOIMPORTER "Build IcoImporter plugin" OFF)
```

Comment:
Would be great if you could enable this in all `*.bat` and `*.sh` file in `package/ci/` so we can see that it builds & runs properly everywhere.

Reply:
ah, forgot about CI!